packages/es-components/src/components/controls/radio-buttons/RadioButton.js

```diff
@@ -68,7 +69,7 @@ const RadioDisplay = styled.span`
   }
 `;
 
-export function RadioButton({ name, children, ...radioProps }) {
+export function RadioButton({ children, ...radioProps }) {
```

Comment:
Why are we removing the `name` prop here? Doing that will be a breaking change.

Reply:
It should just become part of `radioProps`, which is still applied to `RadioInput` as it was before.